src/RVRP_data.jl

```diff
@@ -50,11 +54,10 @@ mutable struct Request # can be
     specific_product_id::String
     split_fulfillment::Bool  # true if split delivery/pickup is allowed, default is false
     precedence_status::Int # default = 0 = product predecessor restrictions;  1 = after all pickups, 2 =  after all deliveries.
-    mantadory_status::Int # default = 0 = mandatory, 1= semi_mandatory (must be covered if a feasible solution exists), 2 = optional
-    reward::Float64 # define if semi_mandatory or optional; reward for fulfilling the request
     product_quantity_range::Range # of the request
-    shipment_capacity_consumption::Vector{Float64} # can include several independant capacity consumptions: as weight, value, volume
-    shipment_property_requirements::Dict{Int,Float64} # to check if the vehicle has the property of accomodating the request: yes if request requirement <= vehicle property capacity for each index referenced requirement
+    request_covering_range::Range # to specify if the request is mandatory, semi_mandatory (must be covered if a feasible solution exists), or optional
+    shipment_capacity_consumption::Dict{String,{Tuple{Float64,Float64}} # to quantify the vehicle capacity that is used for accomodating  lot-sizes of the request along several independant capacity measures whose string id key are in the dictionary: as weight, value, volume; for each such key, the capacity used is the float coef 2 * roundup(quantity /  shipment_lot_size = float coef 1)
```

Comment:
I think `shipment_capacity_consumption` and `shipment_property_requirements` should be in the `SpecificProduct`